src/helm/benchmark/scenarios/contactcenter_convo_summarization_scenario.py

```diff
@@ -0,0 +1,124 @@
+"""
+Scenario for contact center conversational summarization.
+
+Loads input conversations defined in the ContactCenterConversationScenario
+and packs task specific input/output format for summarization tasks.
+
+Task structure
+
+    Conversation:
+            agent: message1
+            visitor: message2
+            ....
+    Summary:
+            summary of the conversation
+
+
+Example from the dataset
+    Conversation: 
+            agent: hi how can i help you today
+            visitor: i need help with my account
+            agent: sure what is your account number
+            visitor: 123456
+            .....
+    Summary:
+           - agent helped visitor with account number 123456
+"""
+
+
+import json
+import os
+from typing import List, Optional
+from helm.benchmark.scenarios.contactcenter_convo_base_scenario import ContactCenterConversationScenario
+from helm.common.general import ensure_file_downloaded, ensure_directory_exists
+from .scenario import Instance, Reference, TEST_SPLIT, CORRECT_TAG, Input, Output
+
+
+class ContactCenterConversationSummarizationScenario(ContactCenterConversationScenario):
+    """
+    Scenario for contact center conversational summarization.
+    """
+
+    name = "cc_convo_summarization"
```

Comment:
Pick a consistent name that is used both here and as the filename. I'd prefer both to be `contact_center_conversation_summarization`, but I'd also be fine with both being `cc_convo_summarization`.

Likewise with the other file.